### DISCUSSION TITLE
Change of back button to reset create game settings

### DISCUSSION THREAD

**Discussion by PhantomFFR** [bafolts/terraforming-mars#5596] [Ideas]
Hi,

I noticed a change in behaviour when using the back button in my browser (Firefox, Windows 10). When going back from a (completed) game to [game creation](https://terraforming-mars.herokuapp.com/new-game), now the settings for a new game get reset. 

I don't like this change as I tend to play multiple single player games in a row. When using the back button to go back to game creation until a few days ago the create game settings were remembered and I could just start a new game immediately without having to click around. That was very convenient. There was virtually no downside to this, as when clicking back once more to return to the main menu and then clicking on "New Game" those settings were reset anyway. So in turn what for me was 3 clicks to start a new game (back twice and "start game") now has become ~12 clicks and a bit of searching around the screen to set u the game options as I like them. 

While I can imagine that this was not intended, it still was really convenient and I wonder if this could be changed back to the former behaviour. Were there any actual issues with the former behaviour that I do not see?

**Comment by kberg** [comment]
This is a long-standing issue: https://github.com/terraforming-mars/terraforming-mars/issues/1871
